lib/Instance.js

```diff
@@ -482,6 +482,14 @@ function Instance(Model, opts) {
 	var addInstanceProperty = function (key) {
 		var defaultValue = null;
 		var prop = Model.allProperties[key];
+		var isEnumerable;
+
+		// Defaults to true unless explicitly set to false
+		isEnumerable = !(
+			prop && prop.hasOwnProperty &&
+			prop.hasOwnProperty('enumerable') &&
+			!prop.enumerable
+		);
```

Comment:
I really like this idea, but it would be better to handle this [at the source](https://github.com/dresende/node-orm2/blob/d596ce606b8fe1aeff57574c591d836e47ee93d0/lib/Property.js#L45) as we do with `required` so that *every* property has `enumerable` set.

Also, this PR needs some tests.